analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -164,9 +165,17 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
                         else -> VcsInfo(type = VcsType(type), url = url, revision = tag)
                     }
                 } else {
-                    // It is a common mistake to omit the "scm:[provider]:" prefix. Add fall-backs for nevertheless
-                    // clear cases.
-                    if (connection.startsWith("git://") || connection.endsWith(".git")) {
+                    val userHostMatcher = USER_HOST_REGEX.matcher(connection)
+
+                    if (userHostMatcher.matches()) {
+                        // Some projects omit the type fragment and use the SCP-like Git URL syntax.
```

Comment:
Instead of "fragment" (URL-speak) I'd use "provider" (Maven-speak) here, similar to the existing comment below.

Reply:
Done.